### PR TITLE
php-http/message stopped providing php-http/message-factory

### PR DESCRIPTION
**Pull request by dbu**
fix https://github.com/php-http/message/pull/152#issuecomment-1550972240

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
         "php-http/client-implementation": "^1.0 || ^2.0",
         "php-http/client-common": "^1.1.0 || ^2.0",
         "php-http/message": "^1.0 || ^2.0",
+	"php-http/message-factory": "^1.0",
```
Comment (jdreesen) [inline composer.json]: Spaces instead of tabs:

```suggestion
        "php-http/message-factory": "^1.0",
```

Comment (fritzmg) [inline composer.json]:
```suggestion
        "php-http/message-factory": "^1.0",
```

**Review by fritzmg — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (dbu) [reply composer.json]: thanks

Comment (dbu) [reply composer.json]: thanks

**Pull request merged**